qt/aqt/editor.py

```diff
@@ -287,7 +288,7 @@ def _addButton(
         self,
         icon: str | None,
         cmd: str,
-        tip: str = "",
+        title_attribute: str = "",
```

Comment:
This is part of the public API, so changing it could break add-ons.

Reply:
Right, I didn't take add-ons into account. I checked only the code base of Anki itself.

Lesson learned, won't happen again. 🙂